cookies/rfc6265bis/prefix/__host.document-cookie.non-secure.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<script src="/cookies/rfc6265bis/resources/testharness.js"></script>
+<script src="/cookies/rfc6265bis/resources/testharnessreport.js"></script>
```

Comment:
These should both be using WPT's `testharness.js` and `testharnessreport.js`, here and in every other HTML file.